Changelog.md

```diff
@@ -1,5 +1,9 @@
+**V1.9.10 - Updates**
+- Fixed a compiler bug when enabling LCD_BUTTON_TEST
```

Comment:
I would also say "compilation issue" rather than "compiler bug"... Thought you found an issue with GCC at first read :laughing:

Reply:
LOL, totally.. must have been a lack of coffee...